packages/agent/conf/examples/oid4vci_metadata/knb_kvk.json

```diff
@@ -0,0 +1,151 @@
+  "metadata": {
+    "credential_issuer": "https://kvk.demo.sphereon.com/issuer",
+    "credential_endpoint": "https://kvk.demo.sphereon.com/issuer/credentials",
+    "display": [
+      {
+        "name": "Kamer van koophandel",
+        "description": "Kamer van koophandel Issuer"
+      }
+    ],
+    "credentials_supported": [
+      {
+        "display": [
+          {
+            "name": "KVK Credential",
+            "description": "KVK Credential",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://ibb.co/5Kjqr3j",
+              "alt_text": "KVK logo"
+            },
+            "background_image": {
+              "url": "https://ibb.co/L6DR6L8",
+              "alt_text": "Crops background"
+            }
+          },
+          {
+            "locale": "en-US",
+            "name": "KVK Credential",
+            "description": "KVK Credential",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://ibb.co/5Kjqr3j",
+              "alt_text": "KVK logo"
+            },
+            "background_image": {
+              "url": "https://ibb.co/L6DR6L8",
+              "alt_text": "Crops background"
+            }
+          },
+          {
+            "locale": "nl-NL",
+            "name": "KVK Credential",
+            "description": "KVK Credential",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://ibb.co/5Kjqr3j",
+              "alt_text": "KVK logo"
+            },
+            "background_image": {
+              "url": "https://ibb.co/L6DR6L8",
+              "alt_text": "Crops background"
+            }
+          }
+        ],
+        "order": [
+          "description",
+          "bedrijdsnaam",
+          "kvknummer",
+          "plaats"
+        ],
+        "credentialSubject": {
+          "description": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "Description"
+              },
+              {
+                "name": "Description",
+                "locale": "en-US"
+              },
+              {
+                "name": "Beschrijving",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "bedrijdsnaam": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "Bedrijdsnaam"
+              },
+              {
+                "name": "CompanyName",
+                "locale": "en-US"
+              },
+              {
+                "name": "Bedrijfsnaam",
+                "locale": "nl-NL"
+              }
+            ]
+          },
+          "kvknummer": {
+            "value_type": "string",
+            "display": [
+              {
+                "name": "KVKNummer"
```

Comment:
See earlier remarks about name everywhere you use KVKNummer. In Dutch this is KVK nummer

Reply:
fixed